coder_sniffer/Drupal/Sniffs/Commenting/DeprecatedSniff.php

```diff
@@ -169,14 +169,17 @@ public function process(File $phpcsFile, $stackPtr)
             }
         } else {
             // The text follows the basic layout. Now check that the versions
-            // match drupal:n.n.n or project:n.x-n.n or project:n.x-n.n-version[n].
+            // match drupal:n.n.n or project:n.x-n.n or project:n.x-n.n-version[n]
+            // or project:n.n.n or project:n.n.n-version[n].
             // The text must be all lower case and numbers can be one or two digits.
             foreach (['deprecation-version' => $matches[1], 'removal-version' => $matches[2]] as $name => $version) {
                 if (preg_match('/^drupal:\d{1,2}\.\d{1,2}\.\d{1,2}$/', $version) === 0
                     && preg_match('/^[a-z\d_]+:\d{1,2}\.x\-\d{1,2}\.\d{1,2}$/', $version) === 0
                     && preg_match('/^[a-z\d_]+:\d{1,2}\.x\-\d{1,2}\.\d{1,2}-[a-z]{1,5}\d{1,2}$/', $version) === 0
+                    && preg_match('/^[a-z\d_]+:\d{1,2}\.\d{1,2}\.\d{1,2}$/', $version) === 0
+                    && preg_match('/^[a-z\d_]+:\d{1,2}\.\d{1,2}\.\d{1,2}-[a-z]{1,5}\d{1,2}$/', $version) === 0
```

Comment:
I think we could collapse those 5 regexes now into just one. Something like this should work:
```php
preg_match('/^[a-z\d_]+:(\d{1,2}\.\d{1,2}\.\d{1,2}|\d{1,2}\.x\-\d{1,2}\.\d{1,2})(-[a-z]{1,5}\d{1,2})?$/', $version)
```